### PR TITLE
Correct minor issues

### PR DESCRIPTION
**Pull request by afern247**
We need to fix docs, add simpler install commands etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +69,9 @@ addresses.forEach((address) => {
     - `null`: The address is valid.
     - `ValidationError.EMPTY_ADDRESS`: The provided address is empty.
     - `ValidationError.INVALID_ADDRESS`: The provided address does not match any supported wallet address patterns.
+
+## Supported Chains
+
+- **EVM (Ethereum Virtual Machine)**
```
Comment (afern247) [inline README.md]: Can you change this to "EVM addresses"

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # road2crypto-wallet-address-validator
 
-A simple utility to validate a cryptocurrency wallet addresses.
+A simple utility to validate cryptocurrency wallet addresses for EVM, Solana, and Bitcoin chains.
```
Comment (afern247) [inline README.md]: You reverted my change, this needs to say: "A simple utility to validate a cryptocurrency wallet address."

File: README.md [hunk 3]
```diff
@@ -12,26 +12,28 @@ npm install road2crypto-wallet-address-validator
...
-import { ValidationError } from "road2crypto-wallet-address-validator/dist/types";
+import isWalletValid from "road2crypto-wallet-address-validator";
 
-const result = checkCryptoAddress("your_wallet_address_here");
+const result = isWalletValid("your_wallet_address_here");
 
-if (result.error) {
-  console.error("Invalid address:", result.error);
+if (!result.valid) {
+  console.error("Error message:", result.error?.message);
```
Comment (afern247) [inline README.md:23]: I'm not sure we should be logging anything here, the error should be handled on the client/backend that uses this util

Comment (afern247) [reply README.md:23]: Sam for other cases

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# wallet-address-validator
+# road2crypto-wallet-address-validator
```
Comment (afern247) [inline README.md:2]: Can you try to extract the validation mechanism from the `web3-validator` package to see if we can get rid of it?

Comment (afern247) [reply README.md:2]: Make sure to keep the same order of wallet validation I had

File: src/index.ts [hunk 5]
```diff
@@ -27,8 +20,8 @@ const walletRegexes: Partial<Record<WalletType, RegExp>> = {
     [WalletType.BITCOIN]: testBitcoin(),
 };
 
-// Validate wallet address function
-const validateWalletAddress = (address: string): WalletType | null => {
+// Function to get wallet address type
+const getWalletAddressType = (address: string): WalletType | null => {
     // Check if the address is a Bitcoin address first
     if (walletRegexes[WalletType.BITCOIN]?.test(address)) {
```
Comment (afern247) [inline src/index.ts]: Can you separate the validation functions and regexes into 1 file? So all EVM, bitcoin and solana regex and validation process would go into 1 file and the index file would just be clean and simple

File: package.json [hunk 6]
```diff
@@ -1,10 +1,11 @@
 {
-    "name": "wallet-address-validator",
+    "name": "road2crypto-wallet-address-validator",
     "version": "1.0.0",
     "description": "A simple utility to validate cryptocurrency wallet addresses for EVM, Solana, and Bitcoin chains.",
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "scripts": {
```
Comment (afern247) [inline package.json:7]: can you also add here the command to publish it, we should be able to do: `npm run publish`

File: README.md (hunk 3 above)
Comment (aahmedfaraz) [reply README.md:23]: This is README file

File: README.md (hunk 1 above)
Comment (aahmedfaraz) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (aahmedfaraz) [reply README.md]: oops, updated

File: README.md (hunk 4 above)
Comment (aahmedfaraz) [reply README.md:2]: Let me see

File: src/index.ts (hunk 5 above)
Comment (aahmedfaraz) [reply src/index.ts]: Updated code with proper folder structure

File: package.json (hunk 6 above)
Comment (aahmedfaraz) [reply package.json:7]: done

File: README.md (hunk 4 above)
Comment (aahmedfaraz) [reply README.md:2]: I tried it, found the function `isAddress` which we need in the package [file](https://github.com/web3/web3.js/blob/4.x/packages/web3-validator/src/validation/address.ts), but as soon as I added the file and the files it needs, then these files also needs other files and its keep going on and on, feels like almost copying the whole code here, I guess we can keep using the package.

File: README.md (hunk 3 above)
Comment (afern247) [reply README.md:23]: Oh I see, I thought it was in the code

File: README.md (hunk 4 above)
Comment (afern247) [reply README.md:2]: that's fine I'll refactor it later, leave it for now

File: src/index.ts [hunk 7]
```diff
@@ -1,68 +1,22 @@
-import { isAddress } from "web3-validator";
-import { ReturnTypeValidationFunction, ValidationError } from "./types";
+// Import necessary modues and types
```
Comment (afern247) [inline src/index.ts]: These kind of comments are redundant

Comment (aahmedfaraz) [reply src/index.ts]: updated

**Review by aahmedfaraz — APPROVED** [review]

**Pull request merged**